docs/pulseload.md

```diff
@@ -62,3 +62,60 @@ ex: <https://community-tc.services.mozilla.com/pulse-messages/>
 
 [pulse guardian]: https://pulseguardian.mozilla.org/whats_pulse
 [yml schema]: https://github.com/mozilla/treeherder/blob/master/schemas/pulse-job.yml
+
+**Here is an example flow you can use:**
+
+## Setup
+
+(These steps should help you set up everything you need to ingest data locally while working on perfherder)
+
+**Create a pulse guardian account and run:**
+
+```bash
+export PULSE_URL=amqp://USER:PASSWORD@pulse.mozilla.org:5671/?ssl=1
+yarn install
+docker-compose up --build
+```
+
+**Run each of these commands in multiple windows:**
+
+```bash
+docker-compose up --build
+docker-compose run -e PROJECTS_TO_INGEST=autoland backend celery -A treeherder worker --concurrency 1
+```
+
+---
+
+**Run the db viewer**
+
+You can use **dbeaver-ce**
```

Comment:
I recommend only mentioning that you can use any database viewer of your choice here. You could mention `dbeaver-ce` in parentheses like `(e.g. dbeaver-ce, mysql workbench, etc.)`